RadixWallet/Features/DappsAndPersonas/Personas/Child/List/PersonaList+Reducer.swift

```diff
@@ -114,17 +115,17 @@ public struct PersonaList: Sendable, FeatureReducer {
 
 	public func reduce(into state: inout State, childAction: ChildAction) -> Effect<Action> {
 		switch childAction {
-		case .persona(id: let id, action: .delegate(.openDetails)):
-			.run { send in
-				let persona = try await personasClient.getPersona(id: id)
-				await send(.delegate(.openDetails(persona)))
+		case let .persona(id, action: .delegate(delegateAction)):
+			switch delegateAction {
+			case .openDetails:
+				.run { send in
+					let persona = try await personasClient.getPersona(id: id)
```

Comment:
what happens with the failure? I suggest adding a catch block on `run` and present the error using errorQueue

Reply:
This was just a refactor to switch over the `delegateAction`, but I will update it to present the error (which I don't know in which cases would the `getPersonas()` throw one).